### DISCUSSION TITLE
v3.2.9

### DISCUSSION THREAD

**Discussion by charabaruk** [PodcastGenerator/PodcastGenerator#723] [Announcements]
# Podcast Generator 3.2.9

Podcast Generator 3.2.9 is now available:

<https://github.com/PodcastGenerator/PodcastGenerator/releases/download/v3.2.9/PodcastGenerator-3.2.9.zip>

This is a patch release for 3.2 containing various bug fixes.

Support for the 3.2.x version series will end one month after release of Podcast Generator 3.3 or any newer non-bugfix version.

## Prerequisites

* PHP 7.0 or newer (8.0 or newer recommended)
* PHP fileinfo extension
* PHP gettext extension
* PHP simplexml and libxml extensions
* Apache 2 or nginx web server

## Installation

See the `INSTALL` or `INSTALL.nginx` files in the root of the zip file for details on how to perform a fresh install of Podcast Generator for Apache 2 or nginx web servers.

### Upgrading from Previous Versions

Upgrading is supported from any release of Podcast Generator 3.1. Directly upgrading from older versions is not supported: If you are running PG 3.0 then you need to first upgrade to 3.1; if you are running PG 2.7 or older you need to upgrade to 3.0 and then to 3.1 before upgrading to PG 3.2.

**Be sure to make a backup of your site before upgrading!**

1. Download `PodcastGenerator-v3.2.9.zip` and extract to a local directory.
2. Upload (or copy) the content of the `PodcastGenerator/` directory to the appropriate location on your web server.
3. Adjust the permissions for the PHP files to `775` if necessary.
4. Open your website in a web browser to kick off the automated upgrade process.
5. Delete the `components/getid3/` directory from your website installation (the getid3 library has been moved to the new `vendor/` directory).

For safety reasons, the automated upgrade process does not remove any existing files, which is why you need to be able to delete `components/getid3/` yourself.

## Changes

See the changelog for a non-exclusive list of changes since the 3.1.x series.

## Contributors

Thanks to everyone who reported bugs and to our translators.

<hr /><em>This discussion was created from the release <a href='https://github.com/PodcastGenerator/PodcastGenerator/releases/tag/v3.2.9'>v3.2.9</a>.</em>

**Comment by kukodageza** [comment]
Hi @charabaruk,
what happened to this release? The file is not available.
Is this repo is now abandoned?
Thanks :)

**Reply by wileycoyote78** [reply]
I believe it is abandoned. The Issues page hasn't been looked at since 2022. The release file for 3.2.8 is available. I stumbled onto this project because my web host offers this CMS and I wanted to play around with it. There seems to be ZERO support, though.

**Reply by charabaruk** [reply]
Sorry, I don't really have much of an excuse for the frozen state of Podcast Generator, especially since the show I used to do went on hiatus some time ago due to real-life commitments. However, the file should be available. You just have to grab it from the Assets section on https://github.com/PodcastGenerator/PodcastGenerator/releases/tag/v3.2.9 -- it _should_ be the same URL as linked above but for some reason GitHub issues a 404 now unless using their own generated link on the releases tag page.

**Reply by wileycoyote78** [reply]
Real-life commitments always seem to get in the way of fun, don't they? Thank for the info.

Upon looking at and comparing the URLs, it appears the problem is the "v" before the version number. The ZIP file name is "PodcastGenerator-3.2.9.zip" but the link in the post is pointing to a file called "PodcastGenerator-**v**3.2.9.zip" so it 404s because that file does not exist. Edit the post and remove the "v" and then the link should work just fine.

I don't suppose you have any docs on how to build a theme file, do you?

**Reply by charabaruk** [reply]
Ok, fixed the link in the post. Unfortunately no, I don't have any docs on building theme files, what you see is what you get. For my own show I just ran a local server and kept tweaking things until they looked the way I wanted, then uploaded the resulting files to the live server.
